content/docs/references/cli/_index.md

```diff
@@ -23,6 +23,8 @@ This document outlines all dellctl commands, their intended use, options that ca
 | [dellctl restore create](#dellctl-restore-create) | Restore an application backup |
 | [dellctl restore delete](#dellctl-restore-delete) | Delete application restores |
 | [dellctl restore get](#dellctl-restore-get) | Get application restores |
+| [dellctl encryption rekey](#dellctl-encryption-rekey) | Rekey an encrypted volume |
+| [dellctl encryption rekey status](#dellctl-encryption-rekey-status) | Get status of a encryption rekey operation |
```

Comment:
Shouldn't it be `rekey-status`, not `rekey status`?

Reply:
yes. fixed.